Backend/app/rider-platform/rider-app/Main/src-read-only/Storage/Beam/TicketBooking.hs

```diff
@@ -30,17 +30,16 @@ data TicketBookingT f = TicketBookingT
     ticketPlaceId :: B.C f Kernel.Prelude.Text,
     updatedAt :: B.C f Kernel.Prelude.UTCTime,
     visitDate :: B.C f Data.Time.Calendar.Day,
-    merchantId :: B.C f (Kernel.Prelude.Maybe (Kernel.Prelude.Text))
+    merchantId :: B.C f (Kernel.Prelude.Maybe Kernel.Prelude.Text)
   }
   deriving (Generic, B.Beamable)
 
 instance B.Table TicketBookingT where
-  data PrimaryKey TicketBookingT f = TicketBookingId (B.C f Kernel.Prelude.Text)
-    deriving (Generic, B.Beamable)
+  data PrimaryKey TicketBookingT f = TicketBookingId (B.C f Kernel.Prelude.Text) deriving (Generic, B.Beamable)
   primaryKey = TicketBookingId . id
 
 type TicketBooking = TicketBookingT Identity
 
-$(enableKVPG ''TicketBookingT ['id] [['personId, 'shortId]])
+$(enableKVPG ''TicketBookingT ['id] [['personId], ['shortId]])
```

Comment:
is it correct change?
for other tables we also have each secondary key as list

https://github.com/nammayatri/nammayatri/blob/main/Backend/app/rider-platform/rider-app/Main/src/Storage/Beam/Booking.hs#L73